pygenometracks/tracksClass.py

```diff
@@ -464,10 +467,21 @@ def parse_tracks(self, tracks_file, plot_regions=None):
                 else:
                     # We set the default value of vhighlight to 0.5
                     track_options['alpha'] = 0.5
+                if 'zorder' in all_keywords:
```

Comment:
Does that really remove the border? I had to apply some weird hacks to achieve this. As far as I remember I had to check the alpha value and set depending on it the alpha value for the border. I need to check my source code again.